src/components/accounts/SetupImplicit.js

```diff
@@ -292,7 +274,7 @@ class SetupImplicit extends Component {
                                 open={whereToBuy}
                             />
                         }
-                        {createAccount &&
+                        {false &&
```

Comment:
@Patrick1904 is this on purpose or debug change?

Reply:
Thanks for catching @vgrichina! Will fix